monitors.py

```diff
@@ -63,6 +65,32 @@ def run(self, env, city: City):
             yield env.timeout(self.f / TICK_MINUTE)
             # self.plot()
 
+class InfectionMonitor(BaseMonitor):
+    def run(self, env, city:City):
+        while True:
+            d = {
+                'store':0,
+                'park':0,
+                'workplace':0,
+                'misc':0,
+                'household':0
+            }
+            for l in d.keys():
+                d[l] = sum([h.is_infectious for h in city.humans if h.location.location_type==l])
```

Comment:
This seems to be counting presence of infectious `Human` at a location. The metric that will be useful will be a break down of where `Human`s got infected i.e. locations where `Human`s caught an infection or got exposed.

Reply:
My bad! I have changed the logic to monitoring the **state change**  of an individual when state changes from:
- is_susceptible->is_exposed
- is_exposed to is_infectious (Considering that a person can be infectious only after having been infected)

Hope this is correct